# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1349244395 14400
#      Wed Oct 03 02:06:35 2012 -0400
# Node ID c8c828276b69a25b708b75f542998919d80f4a55
# Parent  c5749d4eab58224e682291ed5e5f84a6a5c5e9d4
* octave.cc (long_opts): Use --no-jit-compiler to match docs.

diff --git a/libinterp/octave.cc b/libinterp/octave.cc
--- a/libinterp/octave.cc
+++ b/libinterp/octave.cc
@@ -240,17 +240,17 @@ struct option long_opts[] = {
   { "info-program",             required_argument, 0, INFO_PROG_OPTION },
   { "interactive",              no_argument,       0, 'i' },
   { "jit-debug",                no_argument,       0, JIT_DEBUG_OPTION },
   { "line-editing",             no_argument,       0, LINE_EDITING_OPTION },
   { "no-gui",                   no_argument,       0, NO_GUI_OPTION },
   { "no-history",               no_argument,       0, 'H' },
   { "no-init-file",             no_argument,       0, NO_INIT_FILE_OPTION },
   { "no-init-path",             no_argument,       0, NO_INIT_PATH_OPTION },
-  { "no-jit",                   no_argument,       0, NO_JIT_COMPILER_OPTION },
+  { "no-jit-compiler",          no_argument,       0, NO_JIT_COMPILER_OPTION },
   { "no-line-editing",          no_argument,       0, NO_LINE_EDITING_OPTION },
   { "no-site-file",             no_argument,       0, NO_SITE_FILE_OPTION },
   { "no-window-system",         no_argument,       0, NO_WINDOW_SYSTEM_OPTION },
   { "norc",                     no_argument,       0, 'f' },
   { "path",                     required_argument, 0, 'p' },
   { "persist",                  no_argument,       0, PERSIST_OPTION },
   { "quiet",                    no_argument,       0, 'q' },
   { "silent",                   no_argument,       0, 'q' },
